sp800-63-3/sec6_xAL.md

```diff
@@ -173,17 +173,44 @@ The AAL selection does not mean the digital service provider will need to issue
+<div class="text-left" markdown="1">
+<table style="width:100%">
+  <tr>
+    <td><img src="sp800-63-3/media/fal-step1.png" alt="FAL Step 1"/></td>
+  </tr>
+  <tr>
+   <td>Step 1 asks agencies to look at the potential impacts of a federation failure. In other words, what would occur if an unauthorized user could compromise an assertion. Compromise could include being able to replay an assertion to impersonate a valid user or an assertion leaking information through the browser. Risk should be considered from the perspective of the organization and to a valid user, since one may not be negatively impacted while the other could be significantly harmed. Agency risk management processes should commence with this step.</td> 
+  </tr>
+  <tr>
+    <td><img src="sp800-63-3/media/fal-step2.png" alt="FAL Step 2"/></td>
+  </tr>
+  <tr>
 
-Increasing the FAL increases the complexity of the deployment and management of a federation system, as RP keys need to be managed at FAL2 and FAL3, and subscriber keys need to be managed at FAL3. Therefore, RPs should add advanced functionality where it is feasible and warranted for the application.
+   <td>FAL2 is required when any personal information is passed in an assertion. Since the other paths in this decision tree already drive the agency to FAL2 or higher, the question of personal information is only raised at this point. That said, personal information release at all FALs should be considered when performing the risk assessment. FAL2 or higher is required when any personal information is contained in an assertion, as the audience and encryption requirements do not adequately protect personal information from being released. Release of even self-asserted personal information requires assertion protection via FAL2. Even though self-asserted information can be falsified, most users will provide accurate information to benefit from the digital service.</td> 
+
+  </tr>
+  <tr>
+    <td><img src="sp800-63-3/media/fal-step3.png" alt="FAL Step 3"/></td>
+  </tr>
+  <tr>
+   <td>RPs should use a back-channel presentation mechanism as described in [SP 800-63C Section 7.1](sp800-63c.html#back-channel) where possible, particularly at FAL2 or higher, as such mechanisms allow for greater privacy and security. Since the subscriber handles only an assertion reference and not the assertion itself, there is less chance of leakage of attributes or other sensitive information found in the assertion to the subscriber's browser or other programs. As the RP directly presents the assertion reference to the IdP, the IdP can often take steps to identify and authenticate the RP during this step. Further, as the RP fetches the assertion directly from the IdP over an authenticated protected channel, there are fewer opportunities for an attacker to inject an assertion into an RP.</td> 
```

Comment:
FAL1 can't be used with front channel, so strike "particularly at FAL2 or higher".